python/lsst/ts/mtaos/mtaos.py

```diff
@@ -280,6 +283,9 @@ async def configure(self, config: typing.Any) -> None:
         self._logExecFunc()
         self.log.debug("MTAOS configuration started.")
 
+        # Read feature flag early to decide if we start MTPtg remote
+        enable_poc = bool(getattr(config, "enable_pointing_correction", True))
```

Comment:
I would rather you be explicit with the variable name... `enable_poc` and `enable_pointing_correction` "costs" the same amount but the second is a lot clearer...